tutorial/local-setup.md

```diff
@@ -4,90 +4,94 @@ This appendix describes how to check out, build and view the PHP documentation l
 Viewing results as a php.net mirror isn't a simple process, but it can be done.
 The following is one route, and it assumes:
 
-- PHP 5.4+ is available
+- PHP 7.2+, with the following extensions enabled: DOM, libXML2, XMLReader, and SQLite3
 - Git version control system is available
 - A basic level of shell/terminal usage, or know that shell commands follow a `$` below
 
 If you're interested in simply setting up a local PHP mirror (and NOT build the documentation) then
 please follow the php.net [mirroring guidelines](http://php.net/mirroring) and ignore this document.
 
 ## Checkout the php documentation from Git
-**Assumptions**: This assumes using `/tmp` as the root directory, and checkout of the English language.
-Adjust accordingly, including paths, especially if you are on Windows.
+**Assumptions**: A working directory `phpdoc` is created and will contain the necessary cloned repositories. This tutorial will reference the directories that repositories are cloned into. Adjust as needed, including paths, especially if you are on Windows. This tutorial will clone at least three repositories:
+ * doc-en (into the `en` directory)
+ * doc-base 
+ * phd
 
+If setting up a local php.net mirror is desired, the `web-php` repository must also be cloned. This is explained in the [Set up a local php.net mirror](/#set-up-a-local-phpnet-mirror) section.
```

Comment:
```suggestion
If setting up a local php.net mirror is desired, the `web-php` repository must also be cloned. This is explained in the [Set up a local php.net mirror](#set-up-a-local-phpnet-mirror) section.
```

Reply:
I changed it to an HTML hyperlink because I know at least that will work across both markdown and HTML.

Reply:
I'm not sure what "work across both markdown and HTML." means, but the suggested markdown would work fine.

Reply:
My hesitancy with using markdown is not knowing how the Parsedown script works and if it will correctly handle the hyperlink. But this is an issue with my own knowledge and not an issue with the suggestion.

Reply:
Not a problem.  I wouldn't have made the suggestion if I wasn't sure it would work. :)